softgroup/services/jwtService/src/main/java/com/softgroup/token/JwtService.java

```diff
@@ -0,0 +1,56 @@
+package com.softgroup.token;
+
+import com.softgroup.common.dbase.model.ProfileEntity;
+
+import io.jsonwebtoken.*;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+/**
+ * Created by user on 21.03.2017.
+ */
+@Component
+public class JwtService {
+    private byte[] secretKey = new byte[]{1,45,15,12,85};
+
+    public String tokenOut(ProfileEntity profileEntity){
```

Comment:
@werbower Name methods properly! This names provide no information about what method do.